src/Drag.js

```diff
@@ -719,6 +773,17 @@ Drag.STATE = {
 	},
 
 	/**
+	 * Container element for cloned drag placeholder. Only applies if
+	 * `dragPlaceholder` is set to `Drag.Placeholder.CLONE`.
+	 * @type {Element|string}
+	 */
+	cloneContainer: {
+		setter: dom.toElement,
+		validator: 'validateElementOrString_',
+		value: 'body'
```

Comment:
Seeing the demos and the tests, I wonder how one could get back the old behaviour for `cloneContainer`. The way it is defined, one would need to know the `parentContainer` directly.

Maybe we could have `@type {Element|string|boolean}`, so when set to `false`, it will resort back to just appending as a sibling element? Not sure what would happen when set to `true` in that case, though 🤔